gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-bin.zip
```

Comment:
> to the latest version

hmm. @iamdanfox @dansanduleac  I imagine we've seen some instability on 5.5? Is there a ticket with context?

Reply:
@dansanduleac do you have a writeup of the issue you can share? 

TBH i think we're being a bit overly conservative here because we don't need to downgrade anyone that got gradle 5.5 successfully onto develop, as that means it worked.